OpenQA/Benchmark/Stopwatch.pm

```diff
@@ -158,7 +158,7 @@ sub summary ($self) {
 
         my $duration = $event->{time} - $prev_time;
         my $cumulative = $event->{time} - $self->{start};
-        my $percentage = ($duration / $self->total_time) * 100;
+        my $percentage = $self->total_time > 0 ? (($duration / $self->total_time) * 100) : 0;
```

Comment:
hmm , how perl divide by zero? Nice catch.

Reply:
In my test the execution time is basically zero, so I've noticed that issue. In Perl you actually get an exception when trying to divide by zero (in contrast to JavaScript where you get Infinity which is reasonable as well).